timeout@narutrey/info.json

```diff
@@ -1,4 +1,7 @@
 {
-    "author": "narutrey",
+    "author": "",
+    "original-authors": [
+        "narutrey"
+    ],
```

Comment:
This should be:

```json
  "author": "none",
  "original_author": "narutrey"
```

Reply:
Ok but many applets have the  "original-authors" field in their... metadata.json oops.

Reply:
Changes made.

Reply:
The author key should be "none" instead of an empty string.

Reply:
Done